src/store/users/reducer.ts

```diff
@@ -0,0 +1,13 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { fetchCollaboratorsAction } from './actions';
+import { UsersState } from './types';
+
+export const initialState = {
+    collaborators: null,
```

Comment:
Does this need to be nullable or can the default value be an empty array? Do we intend to use `null` to represent a "not fetched yet" state in the UI somewhere?